internal/tart/vm.go

```diff
@@ -94,6 +94,10 @@ func (vm *VM) Start(config Config, gitLabEnv *gitlab.Env) error {
 		runArgs = append(runArgs, "--dir", fmt.Sprintf("hostdir:%s", gitLabEnv.HostDirPath()))
 	}
 
+	for _, customDirectoryMount := range customDirectoryMounts {
+		runArgs = append(runArgs, "--dir", customDirectoryMount)
```

Comment:
Do we need to expand environment variables here so people can pass things like `foo:/Users/admin/foo-$CI_CONCURRENT_ID`?